docs/lifecycle/lifecycle.rst

```diff
@@ -24,12 +24,18 @@ Project States
 +---------------+-------------------------------------------------------------+
 | Project State | State Summary                                               |
 +===============+=============================================================+
-| Non-LFN       | Project does not exist or exists outside of LFN.            |
+| none          | Project does not exist or exists outside of LFN.            |
 +---------------+-------------------------------------------------------------+
-| Non-TAC       | Project is admitted to the LFN but does not have a          |
-|               | representative on the TAC.                                  |
+| Sandbox       | Project is admitted to LFN but does not have direct funding |
+|               | from LFN.  The intent is to enable new projects to gain     |
+|               | visibility and participate in the LFN with minimal impact   |
+|               | on existing projects until they are ready for a subsequent  |
+|               | state.                            			      |	
 +---------------+-------------------------------------------------------------+
-| TAC           | Project is granted TAC representation.                      |
+| Incubation    | Project has matured beyond sandbox and may receive some     |
```

Comment:
I think the consensus was that Incubation projects my receive funding, but not at the expense of TAC projects.

Reply:
I see this is present below in 'Budget Guidance'.  Should we make it explicit here?

Reply:
Yes, I think we could make it explicit here as well

Reply:
added in next commit